src/main/22-vpc_endpoints.tf

```diff
@@ -16,4 +16,26 @@ module "endpoints" {
     }
   }
 
-}
\ No newline at end of file
+}
+
+#SEPARATE BECAUSE NOT FOUND WITH MODULE
+resource "aws_vpc_endpoint" "timestream_ingest" {
```

Comment:
- Add the subnet ids where to create the endpoint, one for each AZ
- Do not use the default SG. Create an SG for these endpoints and allow communication from the EKS pods SG and from BH SG